test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -273,6 +294,45 @@ private static String convertPersonToAddCommandString(ReadOnlyPerson person) {
     }
 
     /**
+     * Test update person command
+     */
+    @Test
+    public void updateCommand_invalidArgs() {
+        final String updateCommandFormatString = "update 1 %s";
+        final String[] inputs = {
+                "update", "update ",
+                "update 1", "update 1 ",
+                "update 0", "update -1 ",
+                String.format(updateCommandFormatString, name),
+                String.format(updateCommandFormatString, name, " "),
+                String.format(updateCommandFormatString, phone),
+                String.format(updateCommandFormatString, phone, " "),
+                String.format(updateCommandFormatString, email),
+                String.format(updateCommandFormatString, email, " "),
+                String.format(updateCommandFormatString, address),
+                String.format(updateCommandFormatString, address, " ")
+        };
+        final String resultMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, UpdateCommand.MESSAGE_USAGE);
+        parseAndAssertIncorrectWithMessage(resultMessage, inputs);
+    }
+
+    @Test
+    public void updateCommand_validArgsParsedCorrectly() {
```

Comment:
Based on our naming convention for test cases, this should be `updateCommand_validArgs_parsedCorrectly`